hcp/io/read.py

```diff
@@ -449,6 +466,8 @@ def _parse_annotations_ica(ica_strings):
             sep = "'"
         val = [(int(ch) if ch.isdigit() else ch) for ch in
                rest.split(sep) if ch.isalnum()]
+        if all(v.isdigit() for v in val):
+            val = [v - 1 for v in val]  # map to Python index
```

Comment:
But htis is the ICA components. The general idea would be to keep things equal by design and add a note on indexing, saying that the outputs get adjusted/manipulated on reading to match Python based expectations.